test/functional/api/api-3rd-party-add.bats

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bats
+
+# Author: Castulo Martinez
+# Email: castulo.martinez@intel.com
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME"
+	create_third_party_repo "$TEST_NAME" 10 staging repo1
+	export repo1="$TPURL"
+
+}
+
+@test "API069: 3rd-party add" {
+
+	run sudo sh -c "echo y | $SWUPD 3rd-party add $SWUPD_OPTS_NO_CERT my-repo file://$repo1 --quiet"
+
+	assert_status_is "$SWUPD_OK"
+	expected_output=$(cat <<-EOM
+		Issuer: /C=US/ST=Oregon/L=Portland/O=Company Name/OU=Org/CN=localhost
```

Comment:
I am unsure if we need to show that if --assume=yes is used, what do you think?

Reply:
hmm you are right, it may be a good idea 🤔

Reply:
Actually my idea was to hide it, but I really don't know. Let's leave how it is.

Reply:
ooh I see... well, let me know what you think